src/clib/lib/enkf/read_summary.cpp

```diff
@@ -16,7 +20,27 @@ static bool matches(std::vector<std::string> patterns, std::string key) {
     }
     return has_key;
 }
+
 ERT_CLIB_SUBMODULE("_read_summary", m) {
+    m.def("read_dates", [](Cwrap<ecl_sum_type> summary) {
+        if (!PyDateTimeAPI) {
+            PyDateTime_IMPORT;
+        }
+
+        time_t_vector_type *tvec = ecl_sum_alloc_time_vector(summary, true);
+        int size = time_t_vector_size(tvec);
+        std::vector<pybind11::object> result(size);
+        auto t = tm{};
+        for (int i = 0; i < size; i++) {
+            auto i_time = time_t_vector_iget(tvec, i);
+            ::gmtime_r(&i_time, &t);
+            result[i] = pybind11::reinterpret_steal<pybind11::object>(
+                PyDateTime_FromDateAndTime(t.tm_year + 1900, t.tm_mon + 1,
```

Comment:
the cpp linter is angry

Reply:
the cpp linter didn't understand the PyDateTime_IMPORT I believe. Added an if in front to placate the linter.